includes/checkout.php

```diff
@@ -315,10 +316,9 @@ function pmprodon_pmpro_checkout_order( $order ) {
 function pmprodon_pmpro_invoice_bullets_bottom( $order ) {
 	$components = pmprodon_get_price_components( $order );
 	if ( ! empty( $components['donation'] ) ) {
-	?>
-	<li><strong><?php _e( 'Membership Cost', 'pmpro-donations' ); ?>: </strong> <?php echo pmpro_formatPrice( $components['price'] ); ?></li>
-	<li><strong><?php _e( 'Donation', 'pmpro-donations' ); ?>: </strong> <?php echo pmpro_formatPrice( $components['donation'] ); ?></li>
-	<?php
+		$bullets = '<li><strong>' . __( 'Membership Cost', 'pmpro-donations' ) . ": </strong> " . pmpro_formatPrice( $components['price'] ) . '</li>' 
+		 . '<li><strong>' . __( 'Donation', 'pmpro-donations' ) . ": </strong>"  .  pmpro_formatPrice( $components['donation'] ) . '</li>';
+	echo $bullets = apply_filters( 'pmpro_donations_bullets_invoice', $bullets, $order );

```

Comment:
Would the filter be easier to use if we set it up like this so that devs can just unset a bullet instead of having to chop up a string?
```suggestion
		$bullets = array(
			'membership_cost' => __( 'Membership Cost', 'pmpro-donations' ) . ": </strong> " . pmpro_formatPrice( $components['price'] ),
			'donation'                => <strong>' . __( 'Donation', 'pmpro-donations' ) . ": </strong>" . pmpro_formatPrice( $components['donation'] )
		);
		apply_filters( 'pmpro_donations_invoice_bullets', $bullets, $order );
		foreach ( $bullets as $bullet ) {
			echo '<li>' . $bullet . '</li>';
		}
```